config/crd/bases/policies.kubewarden.io_clusteradmissionpolicies.yaml

```diff
@@ -288,7 +288,7 @@ spec:
                   ClusterAdmissionPolicy resource.  Known .status.conditions.types
                   are: "PolicyServerSecretReconciled", "PolicyServerConfigMapReconciled",
                   "PolicyServerDeploymentReconciled", "PolicyServerServiceReconciled"
-                  and "PolicyServerWebhookRegistered"'
+                  and "PolicyServerWebhookRegistered" "ClusterAdmissionPolicyActive"'
```

Comment:
`PolicyServerWebhookRegistered` leftover

Reply:
Indeed, missed it before. Fixed it in fixed in https://github.com/kubewarden/kubewarden-controller/pull/76/commits/bded155b91ac3a063fd892ff69e0d0a04f499852.